security/nbde_tang_server_operator/nbde-tang-server-operator-release-notes.adoc

```diff
@@ -14,4 +14,5 @@ link:https://access.redhat.com/errata/RHEA-2023:7491[RHEA-2023:7491]:: The NBDE
 link:https://access.redhat.com/errata/RHEA-2024:0854[RHEA-2024:0854]:: The NBDE Tang Server Operator 1.0.1 has been moved from the `alpha` channel to the `stable` channel.
 link:https://access.redhat.com/errata/RHBA-2024:8681[RHBA-2024:8681]:: The 1.0.2 update contains fixes that increase the Container Health Index of containers deployed with the NBDE Tang Server Operator to the highest grade.
 link:https://access.redhat.com/errata/RHEA-2024:10970[RHEA-2024:10970]:: The 1.0.3 update contains changes that re-increase the Container Health Index to the highest grade.
-link:https://access.redhat.com/errata/RHBA-2025:0663[RHBA-2025:0663]:: With the NBDE Tang Server Operator 1.1, the `golang` package is provided in version 1.23.2 and the `golang.org/x/net/html` package has been updated to version 0.33.0. The updates increase the Container Health Index.
+link:https://access.redhat.com/errata/RHBA-2025:0663[RHBA-2025:0663]:: With the NBDE Tang Server Operator 1.1, the `golang` package is provided in version 1.23.2, and the `golang.org/x/net/html` package has been updated to version 0.33.0. The updates increase the Container Health Index.
```

Comment:
```suggestion
link:https://access.redhat.com/errata/RHBA-2025:0663[RHBA-2025:0663]:: The NBDE Tang Server Operator 1.1 includes `golang` package version 1.23.2 and `golang.org/x/net/html` package version 0.33.0. The updates improve the Container Health Index.
```
Just a suggestion to improve writing style and remove passive voice.

Reply:
```suggestion
link:https://access.redhat.com/errata/RHBA-2025:0663[RHBA-2025:0663]:: The NBDE Tang Server Operator 1.1 includes the `golang` package version 1.23.2 and the `golang.org/x/net/html` package version 0.33.0. The updates improve the Container Health Index.
```

@shreyasiddhartha Looks good. Needs "the" article as added in the above suggestion. "has been updated to..." can be kept to show what's been updated in this RN.